lib/webdriver.py

```diff
@@ -130,7 +129,7 @@ def get_reservations(self, account_monitor: AccountMonitor) -> List[JSON]:
         return reservations
 
     def _get_driver(self) -> Driver:
-        logger.debug("Starting webdriver for current session (this may take a few minutes)")
```

Comment:
Why was this removed? It was added due to #237 and put in both places (this and `Loading Southwest page`) as it may take a while at both places. The message `(this may take a moment)` has better wording than this though.

Reply:
As many tests I have done :) for me it took only a while for the Loading Southwest ... page
The  "Starting webdriver for current session (this may take a few minutes)" always loaded fast. But we can keep that message there as well, I just thought it's not needed.

Reply:
Yeah, that's been my experience too. It can be left off to just say `Starting webdriver for current session`.